src/Core/WinSWCore/Configuration/YamlConfiguration.cs

```diff
@@ -100,6 +100,9 @@ public class YamlConfiguration : IWinSWConfiguration
         [YamlMember(Alias = "securityDescriptor")]
         public string? SecurityDescriptorYaml { get; set; }
 
+        [YamlMember(Alias = "extensions")]
+        public List<string>? YamlExtensionIds { get; set; }
```

Comment:
This one will not work as defined, right? We still need to support extensions somehow

Reply:
Yes. It is my next plan on YAML Support